code/Base/GamemodeSystem/Entities/BaseGamemode.TargetEntities.cs

```diff
@@ -0,0 +1,43 @@
+namespace Sports;
+
+public partial class BaseGamemode
+{
+	public List<GamemodePointEntity> Entities { get; protected set; } = new();
+
+	public IEnumerable<Entity> GetEntitiesWithTag( string tag )
+	{
+		return Entities.Where( x => x.Tags.Has( tag ) );
+	}
+	public IEnumerable<T> GetEntitiesOfType<T>( string tag = null )
+	{
+		if ( tag is null )
+		{
+			return Entities.OfType<T>();
+		}
+		else
+		{
+			return GetEntitiesWithTag( tag )
+				.OfType<T>();
+		}
+	}
+
+	[Event.Entity.PostSpawn]
+	protected virtual void GatherTargetEntities()
+	{
+		var ents = All.OfType<GamemodePointEntity>();
+
+		Log.Info( $"[{Name}] Finding gamemode entities, count: {ents.Count()}" );
+
+		foreach ( var gamemodeEntity in ents )
+		{
+			if ( gamemodeEntity.GamemodeId.ToLower() == Name.ToLower() )
+			{
+				Log.Info( $"	> Matched GamemodePointEntity for {Name}" );
```

Comment:
This too

Reply:
This one is intended so it looks like this
![sbox_Tz2JySglv7](https://user-images.githubusercontent.com/22372616/169662611-a0092e2c-d772-4f07-b7b2-ad70f37ce351.png)

Reply:
Oh you meant log.debug mb